herbert_core/utilities/geometry/bin_inside.m

```diff
@@ -0,0 +1,95 @@
+function is_inside = bin_inside(img_grid_coord,img_size,targ_range,mark_nodes)
+% Found the cells or edges which lie inside the limits provided as input.
```

Comment:
```suggestion
% Find the cells or edges which lie inside the limits provided as input.
```
That was for english.

Based on the definitions of mark_nodes and is_inside for mark_nodes==T, this should be cells or NODES